### DISCUSSION TITLE
Corresponding model with "chinese (simplified)"

### DISCUSSION THREAD

**Discussion by kkckk1110** [MaartenGr/BERTopic#2058] [Q&A]
I am using the following code, but I cannot connect to HuggingFace currently and thus fail to load the transformer model. 
I tried to download the model from the following link but I cannot find the "chinese (simplified)". Can you kindly inform me the corresponding model?

https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/

topic_model = BERTopic(language="chinese (simplified)",calculate_probabilities=False,vectorizer_model=vectorizer
                      ,umap_model=umap_model, low_memory=True) #

**Comment by MaartenGr** [comment]
[This](https://huggingface.co/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2) is the model that will be loaded when you use `language="chinese (simplified"`.

**Comment by kkckk1110** [comment]
Thank you very much!
